10.9-libcxx/stable/main/finkinfo/text/texinfo-dont_build_info.patch

```diff
@@ -0,0 +1,40 @@
+dont build the info documentation, it has been removed
+---
+ doc/Makefile.in |   12 ++++++++----
+ 1 file changed, 8 insertions(+), 4 deletions(-)
+
+--- texinfo.orig/doc/Makefile.in
++++ texinfo/doc/Makefile.in
+@@ -259,13 +259,15 @@
+ am__v_texidevnull_ = $(am__v_texidevnull_@AM_DEFAULT_V@)
+ am__v_texidevnull_0 = > /dev/null
+ am__v_texidevnull_1 = 
+-INFO_DEPS = texinfo.info info-stnd.info texi2any_api.info
++#INFO_DEPS = texinfo.info info-stnd.info texi2any_api.info
++INFO_DEPS =
```

Comment:
Unless I am missing something, this is the only actual change – the others below are already commented out.
Not sure what to make of the "info documentation, it has been removed" comment either; those .info files are built and installed into `%p/share/info`, and without that patch exactly the same number of files is installed.
It seems `$(INFO_DEPS)` is only ever referred to in the `uninstall-info-am` and `maintainer-clean-aminfo` targets, which are probably irrelevant for the Fink build.

Reply:
yeah, that patch is a mess because the original Debian patch comments out INFO_DEPS, TEXINFOS, and info_TEXINFOS, but then @akhansen undid all but INFO_DEPS but in a weird way.
Compare https://raw.githubusercontent.com/debian-tex/texinfo/00a08ac10804f2fc712c3372940e1535850d1eed/debian/patches/dont_build_info and https://raw.githubusercontent.com/fink/fink-distributions/560b8a84237ecd880a58a3d3152704ee9cc93a88/10.9-libcxx/stable/main/finkinfo/text/texinfo-dont_build_info.patch side-by-side. I've left the patch logic as if for now since it's what we have had for 8 years, but I think we might be able to get rid of it since like you said, there's no change in the installed content.